cards.css

```diff
@@ -338,13 +340,13 @@
 .card.modifier.front.left img.cover
 {
     left:                       -25%;
-    clip-path:                  inset(0% 24.5% 0% 24.5%);
+    -webkit-clip-path:                  inset(0% 24.5% 0% 24.5%);
```

Comment:
Are they useful? I tried on Chrome + Safari and both worked just with -webkit-clip-path... but I don't mind having it duplicated!